internal/factsengine/gatherers/corosynccmapctl_test.go

```diff
@@ -8,29 +8,30 @@ import (
 
 	"github.com/stretchr/testify/suite"
 	"github.com/trento-project/agent/internal/factsengine/entities"
-	mocks "github.com/trento-project/agent/internal/factsengine/gatherers/mocks"
+	"github.com/trento-project/agent/internal/factsengine/gatherers"
+	utilsMocks "github.com/trento-project/agent/internal/utils/mocks"
 	"github.com/trento-project/agent/test/helpers"
 )
 
 type CorosyncCmapctlTestSuite struct {
 	suite.Suite
-	mockExecutor *mocks.CommandExecutor
+	mockExecutor *utilsMocks.CommandExecutor
 }
 
 func TestCorosyncCmapctlTestSuite(t *testing.T) {
 	suite.Run(t, new(CorosyncCmapctlTestSuite))
 }
 
 func (suite *CorosyncCmapctlTestSuite) SetupTest() {
-	suite.mockExecutor = new(mocks.CommandExecutor)
+	suite.mockExecutor = new(utilsMocks.CommandExecutor)
 }
 
 func (suite *CorosyncCmapctlTestSuite) TestCorosyncCmapctlGathererMissingFact() {
 	mockOutputFile, _ := os.Open(helpers.GetFixturePath("gatherers/corosynccmap-ctl.output"))
 	mockOutput, _ := io.ReadAll(mockOutputFile)
 	suite.mockExecutor.On("Exec", "corosync-cmapctl", "-b").Return(mockOutput, nil)
 
-	c := NewCorosyncCmapctlGatherer(suite.mockExecutor)
+	c := gatherers.NewCorosyncCmapctlGatherer(suite.mockExecutor)
```

Comment:
I think we could perfectly add this use case in the "happy path" test, but well, it is fine this way too